govc/library/publish.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package library
+
+import (
+	"context"
+	"flag"
+
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	"github.com/vmware/govmomi/vapi/library"
+	"github.com/vmware/govmomi/vapi/rest"
+)
+
+type publish struct {
+	*flags.ClientFlag
+}
+
+func init() {
+	cli.Register("library.publish", &publish{})
+}
+
+func (cmd *publish) Register(ctx context.Context, f *flag.FlagSet) {
+	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
+	cmd.ClientFlag.Register(ctx, f)
+}
+
+func (cmd *publish) Usage() string {
+	return "PATH [ID]..."
+}
+
+func (cmd *publish) Description() string {
+	return `Publish Content Library to subscribers.
+
+Examples:
+  govc library.publish /my-library
+  govc library.publish /my-library id1 id2`
```

Comment:
Might be worth clarifying that these are library subscription ids, rather than library item ids.